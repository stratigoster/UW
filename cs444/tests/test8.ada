PACKAGE foo IS
  j : INTEGER := 10;
  i : INTEGER := 3+4.0*5**(2**j);
--                 ^^^
-- type mismatch: trying to multiply float and integer
END foo;
