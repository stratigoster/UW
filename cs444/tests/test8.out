***ERROR***:3:23:	MUL  works on numeric values of the same type

------
Compilation error
------
